.github/CODEOWNERS

```diff
@@ -1,6 +1,26 @@
 # Instructions for CODEOWNERS file format and automatic build failure notifications:
 # https://github.com/Azure/azure-sdk/blob/main/docs/policies/opensource.md#codeowners
 
+################
+# Orphaned paths
+################
+# As of 1/30/2023 these paths have no owners:
+
+# /
+# /.devcontainer/
+# /documentation/
+# /samples/
+
+###########
+# Eng Sys
```

Comment:
I would like to understandy why these were moved higher and why we didn't remove the ** wildcards here?

Reply:
@weshaggard if I remove the wildcard matches then some pipeline definitions might end up orphaned; no PR reviewers auto-assigned, no build failure notifications sent. Do we want that?

(having said that, I probably should move these paths above `/eng/`).

Reply:
>@weshaggard if I remove the wildcard matches then some pipeline definitions might end up orphaned; no PR reviewers auto-assigned, no build failure notifications sent. Do we want that?

Yes I think we do as most pipelines shouldn't be owned by engsys (outside of the eng folder that is). I wouldn't have expected those wildcards to include any pipelines that are under sdk as they should fallback to the leads instead of EngSys. 

Yes I would keep the /eng/ at the bottom as it isn't as important as the sdks

Reply:
@weshaggard just to clarify:

> Yes I would keep the /eng/ at the bottom as it isn't as important as the sdks

Moving `/eng/` to the bottom will make it the most important - it will match first, before anything else. When `/eng/` was at the top, I saw at least one example of (probably spurious) wildcard match that matched to `/eng/`. I guess we don't want that, so we should keep `/eng/` at the bottom. I understand the idea here is that the set of owners of `/eng/` is disjoint of owners of anything else. So we start with `/eng/` at the bottom, to avoid spurious wildcard matches.

Reply:
I'd be interested to know what matches changed with this move because I wouldn't really expect any.

Reply:
> I'd be interested to know what matches changed with this move because I wouldn't really expect any.

OK this problem is not present in this repo, but in `azure-sdk-for-net`. `RIGHT` is how the owners change when `/eng/` is moved from bottom to the top:

![image](https://user-images.githubusercontent.com/4429827/217418276-fd14d397-c35c-4c41-b61b-8a8e320f2e78.png)

Reply:
OK so just for the record: I reverted the change to `/eng/`; it remains unchanged in this PR, located close to the bottom. I removed `/**/ci.yml` and `/**/tests.yml` instead.